src/test/java/it/pagopa/interop/probing/eservice/operations/integration/repository/EserviceViewRepositoryTest.java

```diff
@@ -36,24 +36,17 @@ class EserviceViewRepositoryTest {
 
 	@BeforeEach
 	void setup() {
-		EserviceView eserviceView = new EserviceView();
-		eserviceView.setEserviceId(UUID.randomUUID());
-		eserviceView.setVersionId(UUID.randomUUID());
-		eserviceView.setEserviceName("e-service Name");
-		eserviceView.setProducerName("Producer Name");
-		eserviceView.setProbingEnabled(true);
-		eserviceView.setVersionNumber(1);
-		eserviceView.setState(EserviceState.ACTIVE);
-		eserviceView.setResponseReceived(OffsetDateTime.parse("2023-03-21T00:00:15.995Z"));
-		eserviceView.setId(10L);
+		EserviceView eserviceView = EserviceView.builder().eserviceId(UUID.randomUUID()).versionId(UUID.randomUUID())
+				.eserviceName("e-service Name").producerName("Producer Name").probingEnabled(true).versionNumber(1)
+				.state(EserviceState.ONLINE).responseReceived(OffsetDateTime.parse("2023-03-21T00:00:15.995Z")).id(10L)
+				.build();
 		testEntityManager.persistAndFlush(eserviceView);
 	}
 
 	@Test
 	@DisplayName("the retrieved list of e-services is not empty")
 	void testFindAll_whenExistsEservicesOnDatabase_thenReturnTheListNotEmpty() {
-		List<EserviceState> listEservice = new ArrayList<>();
-		listEservice.add(EserviceState.ACTIVE);
+		List<EserviceState> listEservice = Arrays.asList(EserviceState.ONLINE);
```

Comment:
Why Arrays to List and not directly a List?

Reply:
Right, I changed